### PR TITLE
Australian Epic for end of year campaign

### PR DESCRIPTION
**Pull request by alexduf**
## What does this change?
This adds an ask at the bottom of the article for our Australian readers.
It will be displayed to 10% of the population, and will help us determine which variant performs best
One of the variants includes a cartoon from First Dog on the Moon. I have embedded it in our `static` folder and I'm using the `config.js` to reference it. Let me know if that approach causes any issue.

## What is the value of this and can you measure success?
We can determine what's the best way to ask for contributions and supporters in Australia, and will use the result of the test to do so.

## Does this affect other platforms - Amp, Apps, etc?
No

## Screenshots
screenshot from 2016-12-14 14-39-48
screenshot from 2016-12-14 14-39-21
screenshot from 2016-12-14 14-38-58
screenshot from 2016-12-14 14-38-27

cc @guardian/guardian-frontend @desbo

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-on-the-moon.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+        function registerInsertionListener(track) {
+            mediator.on(epicInsertedEvent, track);
+        }
+
+        function registerViewListener(complete) {
+            mediator.on(epicViewedEvent, complete);
+        }
+
+        var canBeDisplayed = function() {
```
Comment (jranks123) [inline static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-on-the-moon.js]: This function is not necessary, all the logic around whether the test can run is in the canRun function on line 54.

Comment (alexduf) [inline static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-on-the-moon.js]: Good point! I'm fixing it.

**Comment by alexduf** [comment]
I had another go at it, would you like to check again @jranks123 ?

**Comment by regiskuckaertz** [comment]
Could @paperboyo work his magic and decrease the size of these images?

**Comment by jranks123** [comment]
Looks good from my end 👍

**Comment by paperboyo** [comment]
Yes, Sir! Bit lossy but should be OK.

first-dog-mb_opt2
first-dog-dt_opt3

**Comment by alexduf** [comment]
Here's what I have done:

- Rebased to get the latest test flags so the build can pass
- Added logic to avoid test clash
- fix @jranks123 comments
- replaced images for lighter ones from @paperboyo 

Let me know if there's anything else I should fix

**Comment by jranks123** [comment]
👍

**Pull request merged**